src/services/comments.js

```diff
@@ -32,6 +27,13 @@ module.exports = class ProjectsHelper {
 				})
 			}
 
+			//update the comment
+			if (bodyData.status === common.STATUS_RESOLVED || !bodyData.resolved_by) {
+				bodyData.resolved_by = loggedInUserId
+				bodyData.resolved_at = new Date()
+				bodyData.status = common.STATUS_RESOLVED
+			}
+
 			const filter = {
 				id: comment_id,
 				resource_id: resource_id,
```

Comment:
Pass resource_id first as it is the DC ibn comments table. @priyanka-TL

Reply:
id is the primary key here resource_id is dc but not a part of primary key

Reply:
you should pass the resource_id (distribution column) first to quickly locate the shard, followed by the id (primary key) to quickly locate the specific row within that shard.
This will be more optimised and easy to access if we pass the DC first, as comments table can have huge volume of data we can go for this I guess. 
@priyanka-TL